### PR TITLE
Add University of Bologna as academic institution

### PR DESCRIPTION
**Pull request by boozec**

### CODE REVIEW

File: data/academic_institutions/bologna.md
Comment (cuihtlauac) [inline data/academic_institutions/bologna.md:1]: Hi @boozec, thanks for your contribution, it's great. a `course:` field is missing. This is a list of records with fields: `name`, `acronym`, and `online_resource` of type string, string option and url option (respectively).

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

File: data/academic_institutions/bologna.md
Comment (boozec) [reply data/academic_institutions/bologna.md:1]: Actually, there is no longer any course where OCaml is officially used in the syllabus but there are two courses where OCaml syntax and paradigms are mentioned. Is it still ok?

Comment (cuihtlauac) [reply data/academic_institutions/bologna.md:1]: Yes, this is fine

File: data/academic_institutions/bologna.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: University of Bologna
+description: >
+ The University of Bologna is a public research university in Bologna, Italy. Founded in 1088 by an organised guild of students (studiorum), it is the oldest university in continuous operation in the world, and the first degree-awarding institution of higher learning.
+url: "https://www.unibo.it/"
+logo: academic_institution/bologna.png
+continent: Europe
+courses:
+    - name: Emerging Programming Paradigms
+      acronym: 81672
```
Comment (cuihtlauac) [inline data/academic_institutions/bologna.md]:
```suggestion
      acronym: "81672"
```

File: data/academic_institutions/bologna.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      online_resource: "https://www.unibo.it/en/study/phd-professional-masters-specialisation-schools-and-other-programmes/course-unit-catalogue/course-unit/2023/479059"
+    - name: Logical basis of Computer Science
+      acronym: 30214
```
Comment (cuihtlauac) [inline data/academic_institutions/bologna.md]:
```suggestion
      acronym: "30214"
```

**Review by cuihtlauac — CHANGES_REQUESTED** [review]

**Review by cuihtlauac — APPROVED** [review]
Thanks @boozec. This looks good to me

**Pull request merged**